README.md

```diff
@@ -39,7 +55,15 @@ chaturbate.py     douyutv.py          itvplayer.py   oldlivestream.py  svtplay.p
 
 # Quickstart
 
-We've only just recently forked, but please be patient while we update the code :) (September 17, 2016)
+After installation simply use:
+```
+streamlink twitch.tv/lirik source
```

Comment:
We might not want to endorse any specific broadcaster here to ensure we stay neutral.